composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "require": {
         "php": "^7.1.3",
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^7",
```

Comment:
Is Guzzle 7 compatible with Illuminate 6?

Reply:
just testing some stuff on another branch. This works with --no-scripts but I think there might be another issue with messaging

Reply:
`  Too few arguments to function Illuminate\Database\Migrations\MigrationCreator::__construct(), 1 passed in /app/vendor/xedi/pdk/src/Formation/Migrations/MigrationServiceProvider.php on line 46 and exactly 2 expected
`

Reply:
https://github.com/xedi/pdk-php/pull/439

Reply:
also push slight update so its ^6.3|^7 for guzzle